src/rabbit_peer_discovery_consul.erl

```diff
@@ -551,7 +551,7 @@ start_session_ttl_updater(SessionId) ->
     M = ?CONFIG_MODULE:config_map(?BACKEND_CONFIG_KEY),
     Interval = get_config_key(consul_svc_ttl, M),
     rabbit_log:debug("Starting session renewal"),
-    {ok, TRef} = timer:apply_interval(Interval * 500, ?MODULE,
```

Comment:
The value we have is consistent with https://github.com/rabbitmq/rabbitmq-autocluster/blob/stable/src/autocluster_consul.erl#L132.

Reply:
yes, I am changing the value, I got where is the problem!